contracts/data/RarePizzasSeedStorage.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import '@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol';
+import '@openzeppelin/contracts-upgradeable/utils/math/SafeMathUpgradeable.sol';
+
+import '../interfaces/IChainlinkVRFCallback.0.8.0.sol';
+import '../interfaces/IChainlinkVRFRandomConsumer.0.8.0.sol';
+import '../interfaces/IRarePizzasSeedStorageAdmin.sol';
+
+/**
+ * Contract requests random numbers from VRF and then emits them as events
+ */
+contract RarePizzasSeedStorage is OwnableUpgradeable, IChainlinkVRFCallback, IRarePizzasSeedStorageAdmin {
```

Comment:
this contract gets deployed to matic.  you can skip it for now but its just a storage contract for random numbers associated with job id's.